opentelemetry-instrumentation/README.rst

```diff
@@ -68,10 +68,6 @@ Well known trace exporter names:
 
 ``otlp`` is an alias for ``otlp_span``.
 
-* ``--service-name`` or ``OTEL_PYTHON_SERVICE_NAME``
-
-When present the value is passed on to the relevant exporter initializer as ``service_name`` argument.
```

Comment:
Should we document about providing service_name as part of OTEL_RESOURCE_ATTRIBUTE?

Reply:
That is done here https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py and will also be documented as a part of documenting all supported env vars.